plottr/data/datadict.py

```diff
@@ -87,6 +87,10 @@ def __eq__(self, other: object) -> bool:
                 # print(f"different units for {dn}")
                 return False
 
+            if self[dn].get('label', '') != other[dn].get('label', ''):
+                # print(f"different units for {dn}")
```

Comment:
I guess this should read different label ? These prints should probably all be converted to debug log messages

Reply:
fixed